artview/components/plot_radar.py

```diff
@@ -44,6 +44,7 @@ class RadarDisplay(Component):
     Vgatefilter = None  #: see :ref:`shared_variable`
     VplotAxes = None  #: see :ref:`shared_variable` (no internal use)
     VpathInteriorFunc = None  #: see :ref:`shared_variable` (no internal use)
+    VpyartDisplay = None  #: see :ref:`shared_variable`
```

Comment:
I was under the impression that you were trying to avoid creating a display as a shared variable?